Ports/generate-available-ports.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
```

Comment:
I think `generate-available-ports.py` is better suited to live in `/Meta` instead of `/Ports`, since you do not need it to install or run ports.

Reply:
i was thinking whether put it here or there myself.
- it is used in `Ports` in dev mode—because of that i feel it still belongs there, just as `README.md` generator which just happens to be a part of `./port_include.sh`.
- it seems thereʼs also no call for `./Meta` scripts elsewhere so far—not sure if any place should now about `Meta`…? 🤔

still, i will follow your suggestion here if you believe itʼll be better there; your call? 🙂

Reply:
> it is used in Ports in dev mode

Yeah, it is a meta-functionality but we indeed have README generation in there as well. Let's leave it for now :^)